mne/io/cnt/cnt.py

```diff
@@ -46,14 +46,60 @@ def _read_annotations_cnt(fname, data_format='int16'):
     SETUP_NCHANNELS_OFFSET = 370
     SETUP_RATE_OFFSET = 376
 
+    def _accept_reject_function(keypad_accept):
+        accept_list = []
+        for code in keypad_accept:
+            if 'xd0' in str(code):
+                accept_list.append('good')
+            elif 'xc0' in str(code):
+                accept_list.append('bad')
+            else:
+                accept_list.append('NA')
+        return np.array(accept_list)
+
```

Comment:
I created this function but its functionality could definitely be implemented inside of _update_bad_span_onset. Should I do that or keep it as a separate function?